src/components/Authorization/Authorization.jsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { usePortals } from 'react-portal-hook';
```

Comment:
why do you need modal windows in authorization and registration? If to display an error, then use the snackbars from the "material" library